native/src/main/java/io/ballerina/stdlib/persist/googlesheets/ModuleUtils.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.stdlib.persist.googlesheets;
+
+import io.ballerina.runtime.api.Environment;
+import io.ballerina.runtime.api.Module;
+
+/**
+ * Utility functions relevant to module operations.
+ *
+ * @since 0.1.1
+ */
+public class ModuleUtils {
+
+    private static Module sqlModule;
```

Comment:
```suggestion
    private static Module module;
```

Reply:
fixed in [caecb93](https://github.com/ballerina-platform/module-ballerinax-persist.googlesheets/pull/1/commits/caecb93da85c1d1626d09ea912843b8249d12237)